portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +412,67 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script type="text/javascript">
+
+	var dropdown = document.querySelector('.pagination-items-per-page');
+	var button = dropdown.querySelector('.dropdown-toggle');
+	var list = dropdown.querySelector('.dropdown-menu');
+	var options = list.querySelectorAll('.dropdown-item');
+	var selectedItemValue = button.dataset.attribute;
+
+	button.addEventListener('keydown', function(event) {
+		if (event.key === 'ArrowDown' || event.key === 'ArrowUp' || event.key === 'Enter' || event.key === 'Space') {
+		event.preventDefault();
+		list.classList.add('show');
+		button.setAttribute('aria-expanded', 'true');
+
+			if (list.classList.contains('show')) {
+				var selectedOption = null;
+
+				Array.from(options).find(function(option) {
+					if (option.id == selectedItemValue) {
+						selectedOption = option;
+					}
+				});
+
+				if (selectedOption) {
+					selectedOption.focus();
+				}
+			}
+		}
+	});
+
+	options.forEach(function(option, index) {
+		option.addEventListener('keydown', function(event) {
```

Comment:
Just remember to remove this listener later, as I added in my comment.

Reply:
Fixed